infra/ballista/build/Dockerfile

```diff
@@ -0,0 +1,112 @@
+
+ARG RELEASE_FLAG=release
+ARG FUNCTION_DIR="/function"
+
+
+# build ballista components
+FROM rust:1.63.0-buster as builder
+
+# Include global arg in this stage of the build
+ARG RELEASE_FLAG
+
+ENV RUST_LOG=info
+ENV RUST_BACKTRACE=full
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get -y install \
+    libssl-dev \
+    openssl \
+    zlib1g \
+    zlib1g-dev \
+    libpq-dev \
+    cmake \
+    protobuf-compiler \
+    netcat \
+    curl \
+    unzip \
+    git
+
+# create build directories
+RUN mkdir -p /home/builder/workspace
+
+ENV HOME=/home/builder
+ENV PATH=$HOME/.cargo/bin:$PATH
+
+# prepare rust
+RUN rustup update && \
+    rustup component add rustfmt && \
+    cargo install cargo-chef --version 0.1.34
+
+WORKDIR /home/builder/workspace/
+
+# RUN git clone https://github.com/apache/arrow-ballista.git
+COPY arrow-ballista /home/builder/workspace/arrow-ballista
```

Comment:
this does not work if the user has not manually checked out ballista in the build folder and would not work in [CI](https://github.com/cloudfuse-io/lambdatization/blob/main/.github/workflows/l12n.yaml)

Reply:
your right! I've revert this towards clone on local but just wasn't commited. Is now implemented on f1580e32
I also added the commit or version checkout